runtime/ftplugin/help.lua

```diff
@@ -1,3 +1,7 @@
+-- use treesitter over syntax (for highlighted code blocks)
+vim.treesitter.start()
```

Comment:
`lang = nil` is passed.
I think lang should be `vimdoc`.

Reply:
https://github.com/neovim/neovim/pull/26347#issuecomment-1837508178

It does not work if the language is auto detected.